<launch>
    <!-- <node pkg="usb_cam" name="usb_cam"  type="usb_cam_node" output="log"/> -->

    <node pkg="cv_detection" name="feature_detector" type="feature_detector.py" output="screen">
        <param name="image_topic" type="string" value="/bebop/image_raw" />
        <param name="pub_topic" type="string" value="/control/align_reference/ref_point" />
        <param name="running" type="bool" value="true"/>
    </node>
</launch>
